examples/lombok/sonarqube/build.gradle

```diff
@@ -0,0 +1,11 @@
+plugins {
+    id "org.sonarqube" version "3.0"
+}
+
+apply plugin: "java"
+apply plugin: "io.freefair.lombok"
```

Comment:
I've added a small example project here which applies the lombok and sonarqube plugins in order to test your changes.

The build of this project currently fails, could you please have a look at this?

Reply:
Checked and fixed :)